pkg/crd/persistentip/v1alpha1/doc.go

```diff
@@ -0,0 +1,4 @@
+// +k8s:deepcopy-gen=package,register
+// +groupName=cni.cncf.io
```

Comment:
I'll also be changing this to `k8s.cni.cncf.io` which is what is being used for the other k8snetworkplumbingwg types:
- net-attach-def => https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/blob/93ac89453ffd471cc943eb255799f627777eb1a8/artifacts/networks-crd.yaml#L7
- multi-net policy spec => https://github.com/k8snetworkplumbingwg/multi-networkpolicy/blob/f1873dc329c687c4a2cd236b55284a380b77c6b8/scheme.yml#L7